modules/ocf/manifests/packages/shell.pp

```diff
@@ -0,0 +1,16 @@
+class ocf::packages::shell {
+  package { ['bash', 'tcsh', 'zsh']: }
+
+  file {
+    '/etc/bash.bashrc':
+      source => 'puppet:///modules/ocf/shell/bash.bashrc';
```

Comment:
Oh, I didn't realize bash places something there by default. (Maybe not now, but it could in the future)